### PR TITLE
[ticket/15703] New Template and style for left panel

### PR DESCRIPTION
**Pull request by akbarhashmi**
PHPBB3-15703

Checklist:

- [ ] Correct branch: master for new features; 3.2.x for fixes
- [ ] Tests pass
- [ ] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [ ] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket 

https://tracker.phpbb.com/browse/PHPBB3-15703

### CODE REVIEW

**Comment by akbarhashmi** [comment]
!set WIP :construction:

**Comment by akbarhashmi** [comment]
There still is a LOT to be done including refactoring, proper styling before this can be merged with ucp_pm_viewfolder. I will push more commits soon.

File: phpBB/includes/functions_privmsgs.php [hunk 1]
```diff
@@ -2040,7 +2040,7 @@ function message_history($msg_id, $user_id, $message_row, $folder, $in_post_mode
 	while ($row = $db->sql_fetchrow($result));
 	$db->sql_freeresult($result);
 
-	if (count($rowset) == 1 && !$in_post_mode)
+	if (count($rowset) == 0 && !$in_post_mode)
```
Comment (3D-I) [inline phpBB/includes/functions_privmsgs.php]: `if (!$rowset && !$in_post_mode)` may be?

Comment (akbarhashmi) [reply phpBB/includes/functions_privmsgs.php]: Ah yes ;)

**Comment by VSEphpbb** [comment]
Can you rename PR to say GSOC and that this is PM section.

**Comment by hanakin** [comment]
All should be done in other pr you can close this one

**Pull request closed without merging**